.coveragerc

```diff
@@ -0,0 +1,25 @@
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    # place ``# pragma: no cover`` at the end of a line to ignore it
+    pragma: no cover
+
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if __name__ == .__main__.:
+
+    # `pass` is just a placeholder, fine if it's not covered
+    ^[ \t]*pass$
+
+# Patterns for files to exclude from reporting
+omit =
+    */tests/*
```

Comment:
I wasn't sure if there's still a way to view it through codecov or otherwise (in case someone is interested). I assumed it would be negligible (the code's all getting run either way). So I'll leave it as-is?

Reply:
Yeah that's fair.  I don't think it will show up on codecov either way, but if running locally someone could override the `-omit` option to change the reporting (since all the data will still be there).